public/apps/account/password-reset-panel.tsx

```diff
@@ -64,9 +65,9 @@ export function PasswordResetPanel(props: PasswordResetPanelProps) {
         }),
       });
     } catch (e) {
+      console.error(e);
       setIsCurrentPasswordInvalid(true);
-      setCurrentPasswordError(['Invalid current password due to error: ' + e?.body?.message]);
-      return;
+      setCurrentPasswordError([constructErrorMessage(e, 'Invalid current password.')]);
```

Comment:
Since we need to log the error message everywhere error happens, how about put the console.error into the `constructErrorMessage` and just change the name to like `constructErrorMessageAndLog`. So in the future if we don't want to log to console anymore, it is easier to change.

BTW, we might consider a question: if `e.body.message` is given, do we still need to log to console?

Reply:
Sure. I will add the log to the util function. 

In terms of the full exception, the stack trace on the console may be helpful for troubleshooting. I'd rather keeping logging it even if e.body.message is shown.

Reply:
This is how the console outputs the error.

<img width="2379" alt="Screen Shot 2020-09-17 at 2 47 07 PM" src="https://user-images.githubusercontent.com/60111637/93531878-c93abd00-f8f4-11ea-834c-aa17110dda5d.png">